arbeitszeit_web/www/presenters/remove_worker_from_company_presenter.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.remove_worker_from_company import Response as UseCaseResponse
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+from arbeitszeit_web.www.response import Redirect
+
+
+@dataclass
+class ErrorCode:
+    code: int
+
+
+ViewModel = Redirect | ErrorCode
```

Comment:
just fyi I discussed this with @seppeljordan a little bit in person. To me it seemed strange that you do a type alias with an or here (a union as I learned), only to ensure type hinting is not complaining when an instance of 'ViewModel' is returned. Already the fact that it is even possible in terms of polymorphism (ViewModel can be of type Redirect or ErrorCode and these two classes do not have anything in common!?) confused me. 
But as I learned this is useful later for matching as for example in [register_private_consumption.py](https://github.com/ida-arbeitszeit/arbeitszeitapp/blob/master/arbeitszeit_flask/views/register_private_consumption.py). Maybe we can talk about this again in a meeting. At least for me the explanation by @seppeljordan was very instructive, so thanks again :)